velox/connectors/hive/HiveConnector.cpp

```diff
@@ -62,6 +65,44 @@ HiveConnector::HiveConnector(
           std::make_unique<FileHandleGenerator>(std::move(properties))),
       executor_(executor) {}
 
+std::unique_ptr<DataSource> HiveConnector::createDataSource(
```

Comment:
I dont see this being used anywhere in this PR . Anything basic I am missing ?

Reply:
This is just being moved from the `HiveConnector.h` header to the cpp file.